src/main/java/seedu/recipe/ui/RecipeCard.java

```diff
@@ -143,6 +165,65 @@ public RecipeCard(Recipe recipe, int displayedIndex) {
         });
     }
 
+    private Label createUnorderedListItem(String text) {
+        return createLabel("• " + text);
+    }
+
+    private Label createOrderedListItem(int count, String text) {
+        return createLabel(count + ". " + text);
+    }
+
+    private Label createLabel(String text) {
+        Label label = new Label(text);
+        label.setWrapText(true);
+        return label;
+    }
+
+    private void setIngredients(HashMap<Ingredient, IngredientInformation> ingredientsTable) {
```

Comment:
you may want to abstract out the "No xxx were added for this recipe" message into a constant and use string formatting to generate this notification instead